NuRadioMC/utilities/Veff.py

```diff
@@ -468,38 +222,48 @@ def get_Veff_single(filename, trigger_names, trigger_names_dict, trigger_combina
         fin.attrs['phimin']
     if('phimax' in fin.attrs):
         fin.attrs['phimax']
-    V = fin.attrs['volume']
+    if(veff_aeff == "veff"):
+        volume_proj_area = fin.attrs['volume']
+    elif(veff_aeff == "aeff_surface_muons"):
+        area = fin.attrs['area']
+        # The used area must be the projected area, perpendicular to the incoming
+        # flux, which leaves us with the following correction. Remember that the
+        # zenith bins must be small for the effective area to be correct.
+        volume_proj_area = area * 0.5 * (np.abs(np.cos(thetamin)) + np.abs(np.cos(thetamax)))
```

Comment:
Shouldn't this be np.abs(...) - np.abs(...)  ?

Reply:
no, this calculates the average zenith angle. So it is 1/2*(A+B)

Reply:
Okay, I thought it was supposed to be an integral